src/us/kbase/workspace/kbase/TypeServerMethods.java

```diff
@@ -0,0 +1,107 @@
+package us.kbase.workspace.kbase;
+
+import java.io.IOException;
+import java.net.MalformedURLException;
+import java.util.List;
+import java.util.Map;
+
+import us.kbase.auth.AuthToken;
+import us.kbase.common.service.JsonClientException;
+import us.kbase.common.service.UnauthorizedException;
+import us.kbase.typedobj.exceptions.NoSuchFuncException;
+import us.kbase.typedobj.exceptions.NoSuchModuleException;
+import us.kbase.typedobj.exceptions.NoSuchPrivilegeException;
+import us.kbase.typedobj.exceptions.NoSuchTypeException;
+import us.kbase.typedobj.exceptions.SpecParseException;
+import us.kbase.typedobj.exceptions.TypeStorageException;
+import us.kbase.workspace.FuncInfo;
+import us.kbase.workspace.GetModuleInfoParams;
+import us.kbase.workspace.GrantModuleOwnershipParams;
+import us.kbase.workspace.ListAllTypesParams;
+import us.kbase.workspace.ListModuleVersionsParams;
+import us.kbase.workspace.ListModulesParams;
+import us.kbase.workspace.ModuleInfo;
+import us.kbase.workspace.ModuleVersions;
+import us.kbase.workspace.RegisterTypespecCopyParams;
+import us.kbase.workspace.RegisterTypespecParams;
+import us.kbase.workspace.RemoveModuleOwnershipParams;
+import us.kbase.workspace.TypeInfo;
+
+public interface TypeServerMethods {
+	
+	// TODO JAVADOC
```

Comment:
I kinda feel like javadoc here is a waste of time and code lines since it'd just be copying the docs from `WorkspaceServer`

Reply:
will you just put in a comment to redirect users to the docs for `WorkspaceServer`?

Reply:
yeah, that crossed my mind but I was lazy. You're right though, done

Reply:
Also gave the token variables better names